src/icons.scss

```diff
@@ -441,6 +441,11 @@
       @extend .fa-server;
       @extend .text-danger;
     }
+    &-busy {
```

Comment:
The host is not busy, we are not waiting for it to finish a task, it is disabled.